docs/ReleaseNotes.md

```diff
@@ -3,6 +3,70 @@ id: ReleaseNotes
 title: Release Notes
 ---
 
+## 2020.10.0 - 10/27/2020
+
+### New Features
+
+#### Enklu Embedded
+
+Does your current workflow make you bounce between Unity and the web editor? Have you dreamt of a world where editing an asset in Unity would also edit that asset in the web editor?
+
+Introducing Enklu Embedded in 2020.10! Enklu Embedded allows for users to embed Enklu Cloud content into their Unity scenes. It also enables Unity developers the ability to sync 3D assets from Unity back to Enklu Cloud.
+
+With Enklu Embedded, Unity developers can enjoy the freedom of Unity with the rapid iteration and real-time synchronization of Enklu Cloud (talk about having your cake and eating it too!).
+
+For more information on Enklu Embedded, check out our official guide [here](Embedded/EmbeddedSetup).
+
+
+![Enklu Embedded](/img/product/embedded/VolumeUpdates.gif)
+
+
+
+#### Web Play Mode
+
+New to Enklu Cloud is Web Play Mode! Web Play Mode allows you to test your experience directly from the web editor, without needing to don a headset. You can switch to Web Play Mode via the Play button at the top of the canvas.
+
+Web Play Mode comes in two varieties: Pop-up and Embedded. You can toggle which one you prefer via the Editor Settings tab (Pop-up is enabled by default).
+
+
+![Web Play Mode](/img/product/releases/webplay.gif)
+
+
+### Improvements
+
+#### Sharing Experiences
+
+Adding collaborators to your project should be easier than ever with our new sharing experiences UX! We've added a new Share button next to your user portrait that will allow you to add, remove, and adjust the permissions for all collaborators on a project.
+
+For more tips on editing collaborators and permissions, check out our [official guide](WebEditorBasics/SharingExperiences) on sharing your experiences!
+
+![Sharing Experiences](/img/product/editor/sharing/share_button_and_modal.gif)
```

Comment:
Not a huge deal, but this gif is from before we moved the share button all the way to the right of the presence icons